sky/clouds/aws.py

```diff
@@ -773,6 +755,74 @@ def get_user_identities(cls) -> Optional[List[List[str]]]:
         #   automatic switching for AWS. Currently we only support one identity.
         return [user_ids]
 
+    @classmethod
+    @functools.lru_cache(maxsize=1)  # Cache since getting identity is slow.
+    def get_user_identities(cls) -> Optional[List[List[str]]]:
+        """Returns a [UserId, Account] list that uniquely identifies current AWS
+        principal (user, role or federated identity) whose credentials are used
+        to run current `sky` process. These identities are assumed to be stable
+        for the duration of the `sky` process. Modifying the credentials while
+        the `sky` process is running will not affect the identity returned by
+        this function.
```

Comment:
Style: keep the first line of the docstr to be single line (within 80 characters), we can have detailed descriptions below in a different paragraph.

Reply:
fixed